### ISSUE TITLE
[Screen reader] Remove aria for field_dropdown image element

### ISSUE THREAD

**Issue by microbit-robert** [RaspberryPiFoundation/blockly#9461]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Description

In https://github.com/RaspberryPiFoundation/blockly/pull/9384, `role` and `aria-label` were added to the image element (if applicable) for `field_dropdown`. This causes the dropdown to be read as 'label, image / graphic' rather than providing list-box information and using the aria-label of the parent `g` element. I think [these attributes](https://github.com/RaspberryPiFoundation/blockly/blob/add-screen-reader-support-experimental/core/field_dropdown.ts#L628-L629) should probably be removed so the more informative aria information is used.

### Reproduction steps

1. 
2. 
3.

### Stack trace

```shell

```

### Screenshots

MakeCode examples with VoiceOver below.

If the image has a role and aria-label:
<img width="632" height="280" alt="Image" src="https://github.com/user-attachments/assets/3f07ef28-528a-407d-8007-8688cbb4033f" />

If the image does not have a role and aria-label:
<img width="636" height="279" alt="Image" src="https://github.com/user-attachments/assets/a2c4287c-ca03-4885-bb57-48ad78b249cc" />

**Comment by BenHenning** [comment]
Ah that's interesting. It didn't occur to me that removing the image would actually result in a better experience since it would rely on the item information (which correctly represents the image directly). I completely agree with this change and will send a PR for it momentarily.

**Comment by BenHenning** [comment]
Fixed in #9478.
